paas/install_client.go

```diff
@@ -176,36 +178,34 @@ func (c *InstallClient) showInstallConfiguration(opts *kubernetes.InstallationOp
 func (c *InstallClient) fillInMissingSystemDomain(domain *kubernetes.InstallationOption) error {
 	if domain.Value.(string) == "" {
 		ip := ""
-		if !c.kubeClient.GetPlatform().HasLoadBalancer() {
-			ips := c.kubeClient.GetPlatform().ExternalIPs()
-			if len(ips) > 0 {
-				ip = ips[0]
-			}
-		} else {
-			s := c.ui.Progressf("Waiting for LoadBalancer IP on traefik service.")
-			defer s.Stop()
-			timeout := time.After(2 * time.Minute)
-			ticker := time.Tick(1 * time.Second)
-		Exit:
-			for {
-				select {
-				case <-timeout:
+		s := c.ui.Progressf("Waiting for LoadBalancer IP on traefik service.")
+		defer s.Stop()
+		timeout := time.After(2 * time.Minute)
```

Comment:
I think the code can be simplified if we use this: https://github.com/SUSE/carrier/blob/main/helpers/exec.go#L115

- Start the spinner
- ExecToSuccessWithTimeout
- Handle error or continue

Reply:
I created a new func RunToSuccessWithTimout since ExecToSuccessWithTimeout is used for running process.

Done.